scripts/pylib/twister/twisterlib/runner.py

```diff
@@ -749,6 +749,7 @@ def cleanup_artifacts(self, additional_keep=[]):
         logger.debug("Cleaning up {}".format(self.instance.build_dir))
         allow = [
             os.path.join('zephyr', '.config'),
+            os.path.join('zephyr', 'tfm_merged.hex'),
```

Comment:
I don't understand why this would be added here, and not in the `_get_binaries` function?

Reply:
_get_binaries is reading binary from the runners.yaml, but it is not in the runner.yaml. please check the doc link below
https://docs.zephyrproject.org/latest/samples/tfm_integration/tfm_ipc/README.html

Reply:
And how does this work with sysbuild then since this file will not be in the folder you've hardcoded here?

Reply:
I will manual test it, jsut to help use build server. no sysbuild is used in tfm

Reply:
Well no, sysbuild is not used in tfm itself but sysbuild can and is used to build tfm images

Reply:
canyou help to share the document?

Reply:
it is not working for tfm, below is my steps:

west build --sysbuild -b lpcxpresso55s69_ns

and I check the tfm_ipc folder, the runners.yaml still using the zephyr.hex and not the tfm_merged.hex, and this two files are different.

Reply:
Yes that is expected. The point is the path you have added here `<build_dir>/zephyr/tfm_merged.hex` is wrong when using sysbuild, it would be `<build_dir>/<app_name>/zephyr/tfm_merged.hex`

Reply:
If i do not use sysbuild the file is there.  all the tfm document does not mention using sysbuild